### ISSUE TITLE
Accessibility

### ISSUE THREAD

**Issue by QuincyLarson** [FreeCodeCamp/freecodecamp#159]
# Create '/accessibility-help' url with information pertaining to those that are sight or hearing impaired.

My recommendation is that this page provide additional information for those that use screen readers or require captioning on videos. We can go as in-depth or as basic as necessary.

## Dash Plaintext Transcriptions

We can explain that the Dash challenges are presented as image slides (accessibility FAIL), but that our community has transcribed the slides for easier use in screen readers. *link to Dash transcriptions*

We can also share that for any blind person needing extra assistance with the CSS challenges, that our community is more than willing to help by confirming the visual look or otherwise.\n\n##Did you find an accessibility issue?

Offer the recommended action(s) to take when someone runs into an accessibility issue. Should the camper share this in one of the chat rooms, via Github issues, or email?

---

#Add list item to Challenge 0 that provides accessibility link for campers in need.

This is the first message that openly welcomes everyone with accessibility challenges.

---

# Create page for all Dash challenges

Right now these challenges have been shared with our one known blind camper via gists. I'm assuming there is a better way that we could present these. Any suggestions?

* Also, could the challenges that are still needed be added to the challenge pages as extra credit? It is a lot easier to type this text up as someone is taking the challenge, as opposed to spending 30 minutes to 1 hour typing otherwise (much of that time is spent clicking between slides and such). ESOComputing said it didn't add much time when she was doing it this way. *

**Dash Project 1: Build a Personal Website**

1.1 (still needed)
1.2 (still needed)
1.3 (still needed)
1.4 (still needed)

**Dash Project 2: Build a Responsive Blog Theme**

* 2.1
* 2.2 https://gist.github.com/P1xt/7f43cdd228f20fa205d4
* 2.3 https://gist.github.com/P1xt/a6439c55fd9edac77ee1
* 2.4 https://gist.github.com/P1xt/e80999c769fb8a3f0975

**Dash Project 3: Build a Small Business Website**

* 3.1 https://gist.github.com/brndnb/b35dcf1dbf5fc7a803f7
* 3.2 https://gist.github.com/figitalboy/d51a0a85382c57725665
* 3.3 (still needed)

**Dash Project 4: Build a CSS Robot**

* 4.1 (being transcribed)\n* 4.2 (being transcribed)
* 4.3 (being transcribed)\n* 4.4 (being transcribed)

**Dash Project 5: Build a Madlibs Game**

* 5.1 https://gist.github.com/ESOComputing/103b465a1faf25ce05e1
* 5.2 https://gist.github.com/ESOComputing/0150a0c2a028062ca8fc
* 5.3 https://gist.github.com/ESOComputing/d1a7077599713971b7a1
* 5.4 https://gist.github.com/ESOComputing/d912a7f847795d76ee03
* 5.5 https://gist.github.com/ESOComputing/e473e52f311d968bfeee

---

# Make Challenge Checklist Accessible

Currently, the button that lists all challenges is not accessible. The list itself is fine, but the mechanism of strikethrough on completed challenges does not show up on screen readers. Basically, all challenges "look" the same whether they have been completed or not.

A quick bandaid for this would be to at least add challenge numbers to the user profiles page. Right now, challenges are listed without challenge number. If they are listed with number, a visually-impaired user could check for unfinished challenges by number and go to the challenge list to find them.

Ideally, I think the challenge completion list needs to be a checkbox form.
